iambic/config/wizard.py

```diff
@@ -1012,13 +1024,23 @@ def configuration_wizard_aws_organizations_add(self):
             log.info("Unable to add the AWS Org without creating the required roles.")
             return
 
+        read_only = bool(
+            questionary.confirm(
+                "Do you want to restrict IambicSpokeRole to read-only?\n"
+                "This will limit IAMbic capability to import",
```

Comment:
Same nit about mentioning iambic setup limitation if read_only is true. 

You'll also want to update https://github.com/noqdev/iambic/blob/b374fad63e411e359254d78eeee295de65c083f5/iambic/config/wizard.py#L1697

to 

if (self.config.aws.accounts or self.config.aws.organizations) and not self.config.aws.spoke_role_is_read_only:

Reply:
> Same nit about mentioning iambic setup limitation if read_only is true.
> 
> You'll also want to update
> 
> https://github.com/noqdev/iambic/blob/b374fad63e411e359254d78eeee295de65c083f5/iambic/config/wizard.py#L1697
> 
> to
> 
> if (self.config.aws.accounts or self.config.aws.organizations) and not self.config.aws.spoke_role_is_read_only:

This one may not be a problem because we preserve the secret writing to prefix iambic_config_secret